ansible/roles/operator-pipeline/templates/openshift/tasks/verify-ci-results.yml

```diff
@@ -18,10 +19,13 @@ spec:
       workingDir: $(workspaces.results.path)
       script: |
         #! /usr/bin/env bash
-        # Did the tests pass?
+
         # did preflight/ CI have the latest version?
-        # If results exists, store them in the workspace
-        # In case of failure- comment on the PR with tailored support link
+        PREFLIGHT_VERSION=$(cat PREFLIGHT_RESULTS | jq '.test_library.version' | grep .)
+        if [$PREFLIGHT_VERSION < $(params.preflight_min_version)]; then
+          echo "preflight is having an outdated version"
```

Comment:
Would be good to log what the current version is and what the minimum version is, so it's easier to tell what's wrong.

Reply:
Good Point.